autopts/ptsprojects/stack/layers/bap.py

```diff
@@ -25,8 +25,10 @@ def __init__(self):
     def __init__(self):
         self.peers = {}
         self.ase_configs = []
-        self.broadcast_id = 0x1000000  # Invalid Broadcast ID
+        self.broadcast_id = None
+        self.broadcast_id_2 = None
         self.broadcast_code = ''
+        self.hdl_wid_114 = 0
```

Comment:
```suggestion
        self.hdl_wid_114_cnt = 0
```
Should indicate that this is a counter :)